todo_app/data/trello_items.py

```diff
@@ -0,0 +1,73 @@
+import os
+from datetime import datetime
+from typing import Dict, List, Optional
+
+import requests
+
+from todo_app.data.items import Item, Status
+
+
+class TrelloRequests:
+    def __init__(self):
+        self.errored = False
```

Comment:
Setting an error boolean can be a nice pattern, but worth weighing up the trade-offs against e.g. throwing exceptions - it can be easier to convey extra info with exceptions, and it tends to highlight problems whereas a missed check for `errored` could mean errors are silently ignored; sometimes good and sometimes bad!

Reply:
Hmm yes, throwing exceptions is probably a nicer way of doing this...will have a look